pkg/traceql/ast.go

```diff
@@ -292,6 +288,7 @@ func (a Aggregate) impliedType() StaticType {
 }
 
 func (a Aggregate) extractConditions(request *FetchSpansRequest) {
+	request.AllConditions = false
```

Comment:
shouldn't something like:
```
{ span.foo = "bar" } | sum(duration) > 1s
```
be able to have all conditions true? oh, i bet it can't b/c of nil handling and the fact that we have to pull everything in one pass? "select" we can push into the metadata pass b/c it doesn't do anything except attach the requested cols into the response?

Reply:
This is the same root issue, and kind of undefined behavior (or at least not fully defined).   Does this imply that `duration` must exist or would sum handle nils?  Needs to be generalized for any attribute (in this case duration is a bad example because it always exists).   However for now in this PR, the behavior of this statement hasn't changed.  It was previously AllConditions=false because of the pipeline length check and still is.